uiowa/uiowa
# [Registrar] Correspondence block

Resolves #4965 





# How to test

```
ddev blt ds --site=registrar.uiowa.edu && ddev drush @registrar.local uli /admin/config/sitenow/sitenow-dispatch
```
Update hook should run cleanly
Enter a Dispatch API key and save
```
ddev drush @registrar.local cron
```
Add a page and include the `Correspondence` block
See that there are results!
Check out the filter and see that it properly filters things.
Check the results themselves, that the URL works, etc.
Check that there appear to be 2 years' worth of results



Note: There will be a slight difference in the results for the `Student` and `Faculty/Staff` selections when compared to the existing site. The existing site has extra results not associated with Registrar that should not be appearing. See https://github.com/uiowa/uiowa/issues/4965#issuecomment-2316149199 for details.

Try adding a query parameter to the URL (eg `?audience=student`) and see that it properly sets the filter default on page load

Additional notes:
There will be a follow-up issue for accessibility review/updates and the additional filter mentioned on the issue.
